### ISSUE TITLE
Could not determine current version

### ISSUE THREAD

**Issue by asier-izquierdo** [asier-izquierdo/craftceres#42]
`current_version` determines which version (i.e. 1.21) the server is currently using to compute if there's a new update.

There's being a situation in which, if the server fails to start with the newly downloaded version, it removes the .jar, leaving the server with no .jar, which then causes the script not to be able to determine which version is being used, given that there's none, really.

A possible solution, which needs exploring further consequences if implemented, is to treat the version the same as the build, that is, if none is found, the script simply downloads the latest available, since it may be safe to assume that in running the script, one wants their server updated.

**Comment by asier-izquierdo** [comment]
If `current_version` gets treated the same as `current_build`, which maybe should, there should vary the handling of how the script considers if there is no previous installation, currently does:

```
482 # Or there is no previous installation
483         } || [[ -z "$current_build" ]]
```

But this is a hack, there should be a previous check that if either `current_version`or `current_build` are not found, a variable (maybe called `is_there_anything`) is set to false, and the check should consider that new variable.

Those two variables come from the same file, so one shouldn't be set without the other, but just in case.

**Comment by asier-izquierdo** [comment]
Should be fixed with 07b698315eee1319f7f3f17eaeb1b567ee615b4b, will close when I can verify
